CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 ### Technical Changes
 
+- ROX-18969: Added a label selector for the caching configuration of secrets and configmaps to the operator.
+  - Reduces memory consumption of the operator significantly, especially on large clusters. (28% on a fresh OCP cluster)
+  - Slightly increases requests send to the API server to get secrets that are not managed by the operator and don't match the cache label selector
```

Comment:
```suggestion
  - Slightly increases number of requests sent to the API server to get secrets that are not managed by the operator and don't match the cache label selector.
```
Not sure about the "slightly" part... previously it would be zero requests per reconcile loop in the happy path. Now it will be... a bunch - maybe 3, maybe 7? I don't actually know. In normal state this will be rare (once per 11h or so) but if we get into a retry loop, it might be significant.

Reply:
Removed the "slightly".

I think the happy path after first relabeling should be 4 requests:
- The list for the image pull secrets
- The proxy secret
- Central DB PW
- Central Admin PW

But yes, in a loop without exponential backoff that would probably lead to a lot of requests.